src/compile/legend/encode.ts

```diff
@@ -146,58 +131,13 @@ export function labels(fieldDef: FieldDef, labelsSpec: any, model: UnitModel, ch
   } else if (fieldDef.type === TEMPORAL) {
     labelsSpec = extend({
       text: {
-        signal: timeFormatExpression('datum.value', fieldDef.timeUnit, legend.format, legend.shortTimeLabels, config)
+        // FIXME: shortTimeLabels set as true by default
```

Comment:
Why set to true by default?  Why not just always read from config?  (Modify `timeFormatExpression`?)